server/utils.js

```diff
@@ -0,0 +1,11 @@
+tryParseJson = str => {
+    try {
+        return JSON.parse(str);
+    } catch (e) {
+        return undefined;
```

Comment:
what do you do with the exception? until you have a logger at least output it to the console